packages/edit-post/src/components/layout/style.scss

```diff
@@ -13,49 +13,36 @@
 		color: $dark-gray-900;
 
 		@include break-small {
-			position: fixed;
-			top: inherit;
+			top: 0;
+		}
+
+		// Non-dismissible notices.
+		&.is-pinned.is-pinned {
```

Comment:
Do we need the `.is-pinned` to be duplicated here (specificity)? It'd be preferable if we didn't, or could find another way to override the intended "base" styles.

Reply:
I could swear it was needed, but perhaps it was needed earlier on in the development process, and as things were cleaned up it became unnecessary. In any case I tried removing the extra specificity (yes, cool new trick I learned) and it doesn't seem to have any negative effects.